benchmark/bench/codspeed.js

```diff
@@ -0,0 +1,83 @@
+import { Bench } from 'tinybench';
+import { withCodSpeed } from '@codspeed/tinybench-plugin';
+import { exec } from 'tinyexec';
+import { astroBin } from './_util.js';
+import { waitUntilBusy } from 'port-authority';
+import { benchmarkRenderTime } from './render.js';
+import { benchmarkCannon } from './server-stress.js';
+
+export async function run({ memory, render, stress }) {
+	const bench = process.env.CODE_SPEED ? withCodSpeed(new Bench()) : new Bench();
```

Comment:
```suggestion
	const bench = process.env.CODSPEED ? withCodSpeed(new Bench()) : new Bench();
```

Should we match the env name with `codspeed`?